inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp

```diff
@@ -194,6 +195,35 @@ static void Transformation(CNNNetwork& clonedNetwork, const Config& conf) {
         return false;
     };
 
+    auto isSequencePrimitiveSupported = [](const_node_ptr &node) -> bool {
```

Comment:
Please add comments that explain logic below.

Reply:
fixed